src/components/DisplayNames/index.js

```diff
@@ -95,7 +95,7 @@ class DisplayNames extends PureComponent {
                             <Tooltip
                                 key={index}
                                 text={tooltip}
-                                containerStyle={styles.dInline}
+                                containerStyles={[styles.dInline]}
```

Comment:
```suggestion
                                containerStyle={[styles.dInline]}
```
`containerStyle` is more appropriate. There are no multiple styles. Its fine if it takes array.

Reply:
I think `containerStyles` is better because
- It makes it clearer that we're taking multiple styles (array)
- Is consistent with the rest of the code (we have more components with a `containerStyles` prop)
- Even if we're not adding multiple styles in this specific use of the Tooltip, we might want to it in the future

Reply:
Thanks for clarification. I just checked the style guide. It change so dramatically. LOL. You are correct here.